wordpress-mobile/WordPress-Android
# Add switch to Jetpack in menu items on Phase 4

Part of #17339 

## Changes 

### 1. Swtich to Jetpack menu 
This PR adds a **Switch to Jetpack** menu item to the bottom of the landing page. On clicking the menu item, the overlay should be shown. This PR also add remind me later option for the menu card which will remove the card from the menu for 4 days and will show up on the landing page after that. 

|Landing page in Phase 4 (light) |  Landing page in Phase 4(dark) | 
|---|---|
| ![Screenshot_20230118_152613](https://user-images.githubusercontent.com/17463767/213141887-8117f791-1fba-46f0-b38f-6518f084ce25.png)  |![nightModePhase4](https://user-images.githubusercontent.com/17463767/213142052-36bb32a9-4d31-4fd2-95a0-54d589df3505.png)|

### 2. Phase 4 overlay content 
This PR adds the necessary strings and logic for showing the Phase 4 overlay on clicking at the menu card

|Jetpack phase 4 Overlay Light | Jetpack phase 4 overlay Dark  |
|---|---|
|![lightModePhase4Overlay](https://user-images.githubusercontent.com/17463767/213150717-66cfb2ac-2758-4b06-baa8-a340bb63035d.png)|![nightModePhase4Overlay](https://user-images.githubusercontent.com/17463767/213150737-2fac0ee7-b3e9-4d1d-8cac-31d67a5645d7.png)|


## 👀 Testing Instructions

1. Launch app 
2. Go to app settings -> debug settings -> and enable "jp_removal_four"
3. Verify the menu card is displayed as the last menu item ✅ 
4. Verify tapping the card displays the overlay ✅ 
5. Verify that the Overlay content shown is as per design in the project spec - [Project spec](pe7hp4-4k-p2)
6. Tap the ellipsis and tap on "Remind me later"
7. The card should be removed
8. Wait for the duration to pass - You can change the device date to a date >4 days for testing purposes
9. Pull to refresh the menu
10. Make sure the menu card is displayed


## Regression Notes
1. Potential unintended areas of impact
- Switch to Jetpack card is not displayed as expected 
- Overlay content shown in phase 4 is not correct

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
